### PR TITLE
Merge Rover Campaigns and Rover Experience Repositories

### PR DESCRIPTION
**Pull request by Moux**
Both repositiories are merged into the Rover iOS SDK, with all references to Rover Campaigns removed.

Resolves https://github.com/judoapp/rover-issues/issues/14

### CODE REVIEW

File: make-release.sh [hunk 1]
```diff
@@ -72,4 +74,4 @@ git push origin develop
 
 git push origin v$VERSION
 
-echo "Now run 'pod trunk push Rover.podspec'"
```
Comment (orospakr) [inline make-release.sh:75]: This script should be removed entirely.

File: Cartfile [hunk 2]
```diff
@@ -0,0 +1 @@
+github "RoverPlatform/rover-ios" ~> 4.0
```
Comment (orospakr) [inline Cartfile]: this file should be removed entirely

File: Sources/Experiences/ExperiencesAssembler.swift [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+        
+        container.register(UIViewController.self, name: "experience", scope: .transient) { (resolver, universalLink: URL, campaignID: String?, screenID: String?) in
+            let viewController = RoverViewController()
+            viewController.loadExperience(universalLink: universalLink, campaignID: campaignID, initialScreenID: screenID)
+            return viewController
+        }
+        
+        // MARK: Analytics
+        //TODO: adjust analytics to match the rest of the SDK
+        Analytics.shared.enable()
```
Comment (orospakr) [inline Sources/Experiences/ExperiencesAssembler.swift]: this is better moved to containerDidAssemble() below so this side-effect bit of setup is done after the entire container is injected.

File: Example/Example/AppDelegate.swift [hunk 4]
```diff
@@ -2,169 +2,177 @@
...
-            
-            // Instantiate a `RoverViewController` and call the `loadExperience(id:campaignID)` method to load the
-            // experience whose id matches the value passed in the query parameter.
-            let roverViewController = RoverViewController()
-            roverViewController.loadExperience(id: id, campaignID: campaignID, initialScreenID: screenID)
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Initialize the Rover SDK with all modules.
+        RoverFoundation.initialize(assemblers: [
+            FoundationAssembler(),
+            DataAssembler(accountToken: "be40ec7dabf7c9a06a451e58a4c39d9823c24974"),
```
Comment (orospakr) [inline Example/Example/AppDelegate.swift]: you committed your token by mistake here

**Review by orospakr — CHANGES_REQUESTED** [review]

File: Example/Example/AppDelegate.swift (hunk 4 above)
Comment (Moux) [reply Example/Example/AppDelegate.swift]: Ooops.  Fixed.

File: Sources/Experiences/ExperiencesAssembler.swift (hunk 3 above)
Comment (Moux) [reply Sources/Experiences/ExperiencesAssembler.swift]: Fixed.

File: Cartfile (hunk 2 above)
Comment (Moux) [reply Cartfile]: Fixed.

File: make-release.sh (hunk 1 above)
Comment (Moux) [reply make-release.sh:75]: Fixed.

**Review by orospakr — APPROVED** [review]

**Pull request merged**